src/models/models.py

```diff
@@ -182,4 +182,4 @@ def forward(self, x, y):
     def tmul(self, x, y):  # Takes (I * J * K)(K * L) -> I * J * L
         x_shape = x.size()
         y_shape = y.size()
-        return torch.mm(x.view(-1, x_shape[2]), y).view(-1, x_shape[1], y_shape[1])
+        return torch.mm(x.reshape(-1, x_shape[2]), y).reshape(-1, x_shape[1], y_shape[1])
```

Comment:
what's the reason for this change? just out of curiousity

Reply:
when using the view function, it will give error with the onnx inference, the error message is like below:
`RuntimeError: view size is not compatible with input tensor's size and stride (at least one dimension spans across two contiguous subspaces). Use .reshape(...) instead.`